client/cli/src/runner.rs

```diff
@@ -116,11 +116,18 @@ impl<C: SubstrateCli> Runner<C> {
 	/// Create a new runtime with the command provided in argument
 	pub fn new<T: CliConfiguration>(cli: &C, command: &T) -> Result<Runner<C>> {
 		let tokio_runtime = build_runtime()?;
+		let runtime_handle = tokio_runtime.handle().clone();
 
 		let task_executor = {
```

Comment:
Looks like this block is not useful anymore

Reply:
Not sure I follow here. Which block are you referring to? :)

Reply:
Line 121 on the right ^_^

Reply:
Sharp eyes! Thanks :)